engines/asylum/.gitignore

```diff
@@ -0,0 +1,3 @@
+.deps
```

Comment:
These entries are already covered by the global .gitignore in the project root folder.